src/Aquifer.Jobs/appsettings.json

```diff
@@ -28,19 +28,20 @@
+            "RUS": "Always replace quotes with proper usage of «». Take special care to make the Russian sound natural, using correct sentence structures and Biblical terminologies for Russian. Make sure to always use proper Russian punctuation.",
+            "ZHS": "Use full shape for all punctuation (e.g. '、' for ',', and '）' for ')'. Always replace quotes with proper usage of 「」. When translating any text that looks like a reference to a Bible verse or Bible passage do not put a space between the book and the number. When translating a list of html span elements that have Bible verses or passages inside them, always replace delimiting characters (e.g. ';', ',', '.') such that any list of spans with Bible chapters is delimited by '，', and any list of spans with Bible verses is delimited by '、'. For example, 'cp. 1 Kgs 1:31, 33, 2:1, 2 Ch 33:14' should translate to '参王上1:31、33，2:1；代下33:14', 'see Job 18:6, 18; 19:1' should translate to '见伯18:6、18，19:1', and 'Pss 19:1-6; 97:11; 104:19-20; Isa 60:19-20' should translate to '詩19:1-6，97:11，104:19-20；賽60:19-20'.",
+            "ZHT": "Use full shape for all punctuation (e.g. '、' for ',', and '）' for ')'. Always replace quotes with proper usage of 「」. When translating any text that looks like a reference to a Bible verse or Bible passage do not put a space between the book and the number. When translating a list of html span elements that have Bible verses or passages inside them, always replace delimiting characters (e.g. ';', ',', '.') such that any list of spans with Bible chapters is delimited by '，', and any list of spans with Bible verses is delimited by '、'. For example, 'cp. 1 Kgs 1:31, 33, 2:1, 2 Ch 33:14' should translate to '參王上1:31、33，2:1；代下33:14', 'see Job 18:6, 18; 19:1' should translate to '見伯18:6、18，19:1', and 'Pss 19:1-6; 97:11; 104:19-20; Isa 60:19-20' should translate to '詩19:1-6，97:11，104:19-20；賽60:19-20'."
         },
+        "PlainTextTranslationPromptFormatString": "You receive a string and then return that string in the {0} language. Translate the exact string that you receive. Do not interpret it as anything other than the exact string that it is.",
         "Temperature": 0.2,
-        "TextTranslationPromptFormatString": "You receive a string and then return that string in the {0} language. Translate the exact string that you receive. Do not interpret it as anything other than the exact string that it is."
+        "TextImprovementPromptFormatString": "Improve the grammar and clarity of the provided {0} text. Never echo the original content, only respond with the alteration.",
+        "TranslationPromptFormatString": "Never echo the original content, only respond with the translation. Then translate to {0}."
```

Comment:
Or maybe even "only respond with the alteration" is better here too.

Reply:
The current ordering in this PR is consistent with the way that they were previously because the old [HTML translation base prompt](https://github.com/BiblioNexusStudio/aquifer-server/blob/a5aae8cbfb5345a548d1b6077cacab708e227e48/src/Aquifer.Jobs/appsettings.json#L33) was followed by the [default translate prompt](https://github.com/BiblioNexusStudio/aquifer-server/blob/a5aae8cbfb5345a548d1b6077cacab708e227e48/src/Aquifer.Jobs/appsettings.json#L31).

I split out the "HTML" part so that it could be used for both translation and text improvement (including Aquiferization where it was previously mostly copy/pasted).

Do you still want to swap them?

Reply:
I went ahead [and swapped them](https://github.com/BiblioNexusStudio/aquifer-server/pull/589/commits/c3ca98abdac57da8f10bcf461a7b5930d0140adb) because that's consistent with the text improvement prompt.